tox.ini

```diff
@@ -23,13 +23,13 @@ commands =
 
 [testenv:pep8]
 commands =
-    pep8 flaky
-    pep8 test
+    pep8 --ignore=E501 flaky
+    pep8 --ignore=E501 test
 
 [testenv:pylint]
 commands =
     pylint --rcfile=.pylintrc flaky
-    pylint --rcfile=.pylintrc test -d C0330
+    pylint --rcfile=.pylintrc test -d C0330,W0621
```

Comment:
That's the redefined outer name warning that has to be disabled every time you use a pytest fixture from within the same module where it's defined.